packages/apollo-voyager-keycloak/src/schemaDirectives/hasRole.test.ts

```diff
@@ -0,0 +1,249 @@
+import test from 'ava'
+
+import {GraphQLSchema} from 'graphql'
```

Comment:
Really minor thing but just check spacing around imports like this. I'm wondering is this something your IDE does? I wonder can we lint for this.

Reply:
Yeah, my IDE does format like that.
But I was assuming it is using eslint. Let me have a look at this ...tomorrow.

Reply:
@darahayes I tried tslint's --fix now and it doesn't fix the spaces for that case.
I tried bunch of tslint rules to fix/enforce that behavior but I wasn't successful. 

UPDATE: I found this open issue on tslint: https://github.com/palantir/tslint/issues/1044

Anyway, I will fix this manually.